### PR TITLE
Code Cleaning

### PR DESCRIPTION
**Pull request by AurevoirXavier**
Restruct the cli part args for the future service refactor.

<sup> Created from VS Code using [CodeStream](https://codestream.com/?utm_source=cs&utm_medium=pr&utm_campaign=github*com)</sup>

### CODE REVIEW

File: bin/node/cli/src/cli.rs [hunk 1]
```diff
@@ -62,13 +62,9 @@ pub struct RunCmd {
...
 	pub authority_discovery_disabled: bool,
 
-	/// Maximum number of logs in a query.
-	#[structopt(long, default_value = "10000")]
-	pub max_past_logs: u32,
-
-	/// The dynamic-fee pallet target gas price set by block author
-	#[structopt(long, default_value = "1000000000")]
-	pub target_gas_price: u64,
+	#[allow(missing_docs)]
```
Comment (AsceticBear) [inline bin/node/cli/src/cli.rs:65]: Remove this allow and add some documentation instead.

File: bin/node/cli/src/cli.rs [hunk 2]
```diff
@@ -117,3 +113,14 @@ pub enum Subcommand {
...
+#[derive(Debug, StructOpt)]
+pub struct DynamicFeeParameters {
+	/// Maximum number of logs in a query.
+	#[structopt(long, default_value = "10000")]
+	pub max_past_logs: u32,
+
+	/// The dynamic-fee pallet target gas price set by block author
+	#[structopt(long, default_value = "1000000000")]
+	pub target_gas_price: u64,
+}
```
Comment (AsceticBear) [inline bin/node/cli/src/cli.rs:126]: Actually, `max_past_logs` has nothing to do with DynamicFee. I think to keep the old way would be better.

File: bin/node/cli/src/cli.rs (hunk 1 above)
Comment (AurevoirXavier) [reply bin/node/cli/src/cli.rs:65]: This is a flattened field. The user can not see this from the cli.

File: bin/node/cli/src/cli.rs (hunk 2 above)
Comment (AurevoirXavier) [reply bin/node/cli/src/cli.rs:126]: Rename to `DvmArgs`.

Comment (AsceticBear) [reply bin/node/cli/src/cli.rs:126]: Still not push yet?

File: bin/node/cli/src/cli.rs [hunk 3]
```diff
@@ -62,13 +62,9 @@ pub struct RunCmd {
...
-	/// Maximum number of logs in a query.
-	#[structopt(long, default_value = "10000")]
-	pub max_past_logs: u32,
-
-	/// The dynamic-fee pallet target gas price set by block author
-	#[structopt(long, default_value = "1000000000")]
-	pub target_gas_price: u64,
+	#[allow(missing_docs)]
+	#[structopt(flatten)]
+	pub dynamic_fee_parameters: DvmArgs,
```
Comment (AsceticBear) [inline bin/node/cli/src/cli.rs:67]: pub dvm_args: DvmArgs,

**Review by AsceticBear — CHANGES_REQUESTED** [review]

**Review by HackFisher — APPROVED** [review]

**Pull request merged**